daemon/src/taker_cfd.rs

```diff
@@ -249,7 +374,62 @@ impl Actor {
         }
         Ok(())
     }
+}
+
+impl<O, M> Actor<O, M>
+where
+    O: xtra::Handler<oracle::MonitorAttestation>,
+    M: xtra::Handler<monitor::StartMonitoring>,
```

Comment:
I think that is the only message we are sending. Should we use a `MessageChannel` here?